python/GafferSceneUI/InstancerUI.py

```diff
@@ -306,6 +640,142 @@
 
 		],
 
+
+		"generateSeeds" : [
+			"description",
+			"""
+			Create a seeds context variable based on the id primvar.  This hashes the point id
```

Comment:
It might be good to add `instanceID` to `OSLObject` as per #4092.
Is it worth noting too that if no instanceID is available, each point will be given an ID automatically?

Reply:
Sounds good, should we just do that as part of #4092?